common/federation/src/main/java/app/coronawarn/server/common/federation/client/CloudFederationFeignHttpClientProvider.java

```diff
@@ -34,45 +32,44 @@
    *
    * @param config .
    */
-  public CloudFederationFeignHttpClientProvider(FederationGatewayConfig config,
-      HostnameVerifierProvider hostnameVerifierProvider) {
+  public CloudFederationFeignHttpClientProvider(FederationGatewayConfig config) {
     var ssl = config.getSsl();
     this.connectionPoolSize = config.getConnectionPoolSize();
     this.keyStore = ssl.getKeyStore();
-    this.keyStorePassword = ssl.getKeyStorePass();
-    this.hostnameVerifierProvider = hostnameVerifierProvider;
+    this.keyStorePassword = ssl.getKeyStorePassword();
     this.trustStore = ssl.getTrustStore();
     this.trustStorePassword = ssl.getTrustStorePassword();
   }
 
   /**
    * Creates a FeignClient.
    */
+  @Override
   @Bean
   public Client createFeignClient() {
     return new ApacheHttpClient(
-        federationHttpClientFactory(connectionPoolSize, keyStore, keyStorePassword)
-            .createBuilder().build());
+        federationHttpClientFactory(connectionPoolSize, keyStore, keyStorePassword).createBuilder().build());
   }
 
   /**
    * Creates an {@link ApacheHttpClientFactory} that validates SSL certificates but no host names.
    */
   private ApacheHttpClientFactory federationHttpClientFactory(int connectionPoolSize, File keyStorePath,
       String keyStorePass) {
-    return new DefaultApacheHttpClientFactory(HttpClientBuilder.create()
-        .setMaxConnPerRoute(connectionPoolSize)
-        .setMaxConnTotal(connectionPoolSize)
+    return new DefaultApacheHttpClientFactory(HttpClientBuilder.create() //
```

Comment:
Would you like to add some comments here? If not, can these be removed?

Reply:
These line-end comments force formatting. Can be removed if you want...?